zkevm-circuits/src/test_util.rs

```diff
@@ -8,6 +8,12 @@ use halo2_proofs::dev::{MockProver, VerifyFailure};
 use mock::TestContext;
 use pairing::bn256::Fr;
 
+#[cfg(test)]
+#[ctor::ctor]
+fn init_env_logger() {
+    // Enable RUST_LOG during tests
+    env_logger::init();
+}
```

Comment:
I understand the concerns here!
For the first point, this would be a dev feature right?  I don't think rust supports dev features though (I don't think we should expose a public feature that starts the env_logger, as that's a decision that must be made by the consumer of the library).  Nevertheless, for the concern about having lots of outputs by default, we could mitigate this by setting the default log level to `error`, and then via the `RUST_LOG` env var you can dynamically enable lower levels.

For the second point, I find the usage of `ctor` interesting, because it guarantees that `env_logger::init()` is called once always in tests at the beginning, without requiring updating the tests themselves.  Using `lazy_static` would require that every test function calls a setup function (or use a crate that handles test setup, but in any case we still need to add 1 line for each test function, which is cumbersome).